### PR TITLE
Buzzer module

### PR DESCRIPTION
**Pull request by DanielHernandez-0801**

### CODE REVIEW

File: firmware/src/buzzer.c [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    return CLOCK_OK;
+}
+
+ClockStatus Buzzer_SetVolume(Buzzer *self, uint8_t vol)
+{
+    if (vol < MIN_VOLUME || vol > MAX_VOLUME) {
+        return CLOCK_FAIL;
+    }
+
+    (self->is_active) ? self->stopPwm() : 0;
```
Comment (Matt-Lemcke) [inline firmware/src/buzzer.c]: Does this work the same way as an if statement?

File: firmware/inc/buzzer.h [hunk 2]
```diff
@@ -0,0 +1,31 @@
+#ifndef FIRMWARE_INC_BUZZER_H_
+#define FIRMWARE_INC_BUZZER_H_
+
+#include <stdint.h>
+#include <stdbool.h>
+#include <stddef.h>
```
Comment (Matt-Lemcke) [inline firmware/inc/buzzer.h:6]: What are you using from this library?

**Review by Matt-Lemcke — COMMENTED** [review]
Just a couple questions. Otherwise looks good

File: firmware/src/buzzer.c (hunk 1 above)
Comment (DanielHernandez-0801) [reply firmware/src/buzzer.c]: Yes

File: firmware/inc/buzzer.h (hunk 2 above)
Comment (DanielHernandez-0801) [reply firmware/inc/buzzer.h:6]: NULL

File: firmware/src/buzzer.c (hunk 1 above)
Comment (Matt-Lemcke) [reply firmware/src/buzzer.c]: Is there a performance reason for writing it like this? If you are wanting to keep it to one line I think 
`if(self->is_active){self->stopPwm();}` is more readable

Comment (DanielHernandez-0801) [reply firmware/src/buzzer.c]: No performance reason, just readability purposes. Cool, will make this change.

**Review by Matt-Lemcke — APPROVED** [review]
Looks good

**Pull request merged**